containerd-shim/kata/create.go

```diff
@@ -0,0 +1,257 @@
+// Copyright (c) 2014,2015,2016 Docker, Inc.
+// Copyright (c) 2017 Intel Corporation
+// Copyright (c) 2018 HyperHQ Inc.
+//
+// SPDX-License-Identifier: Apache-2.0
+//
+
+package kata
+
+import (
+	"fmt"
+	"strings"
+
+	"context"
+	vc "github.com/kata-containers/runtime/virtcontainers"
+	vf "github.com/kata-containers/runtime/virtcontainers/factory"
+	"github.com/kata-containers/runtime/virtcontainers/pkg/oci"
+	"github.com/opencontainers/runtime-spec/specs-go"
+	"github.com/sirupsen/logrus"
+)
+
+func create(ctx context.Context, s *service, containerID, bundlePath, netns string, detach bool,
```

Comment:
Does this context contain an opentracing span do you know (https://godoc.org/github.com/opentracing/opentracing-go)? I suspect that the answer is either "no" or "we don't know"). It would be good to be able to enable tracing for the containerd-shim as we now have for the runtime (#565, #617, #641). But since the containerd-shim is not creating the context I suspect we'll have to create our own spans in a later PR.

Reply:
Hi @jodh-intel 
Since containerd-shim hasn't support opentracing,  If we put the kata shimv2 into containerd repo at last, I don't think  they can accept adding opentracing span there.